pom.xml

```diff
@@ -99,12 +100,32 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
-	<!--TODO: Move transport related dependencies here pending https://jira.spring.io/browse/XD-3337  -->
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-stream</artifactId>
 		</dependency>
+		<!-- Cloud connector dependencies -->
+		<!-- Lattice connector dependency to create services info from lattice -->
+		<dependency>
```

Comment:
That probably does make sense for programmatic composition where modules are embedded, and if they are being embedded into a new module, that itself could (at the developer's discretion) use the parent POM. @mbogoevici  wdyt?